### PR TITLE
example_text_completion: only profile first warm run

### PR DESCRIPTION
**Pull request by cota**

### CODE REVIEW

**Review by ManfeiBai — APPROVED** [review]
LGTM, thanks!

File: example_text_completion.py [hunk 1]
```diff
@@ -56,8 +56,8 @@ def main(
 #        cheese =>""",
     ]
     for i in range(repeat):
-        # Automatically takes profiles, let's skip the cold run and only capture warm runs.
-        if i > 0 and not USE_CUDA and xm.is_master_ordinal():
+        # Automatically takes profiles. Let's skip the cold run and only capture the first warm run.
```
Comment (alanwaketan) [inline example_text_completion.py]: Depends on the profile duration, it could capture more runs.

I would suggest probably no changes to the comment.

Comment (cota) [reply example_text_completion.py]: Done, thanks

**Review by alanwaketan — APPROVED** [review]
LGTM.

**Comment by ManfeiBai** [comment]
Hi, there is failure like this after rebase: https://gist.github.com/ManfeiBai/90a18d5700f65695b98388977d90a94d

**Comment by alanwaketan** [comment]
> Hi, there is failure like this after rebase: [gist.github.com/ManfeiBai/90a18d5700f65695b98388977d90a94d](https://gist.github.com/ManfeiBai/90a18d5700f65695b98388977d90a94d)

The failure is showing that you have another process occupying TPU...

**Pull request merged**